provider/sourcegraph-search/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@openctx/provider-sourcegraph-search",
+  "version": "0.0.1",
+  "description": "Sourcegraph Search (OpenCtx provider)",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sourcegraph/openctx",
+    "directory": "provider/sourcegraph-search"
+  },
+  "type": "module",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "index.ts",
+    "!**/*.test.*",
+    "README.md"
+  ],
+  "sideEffects": false,
+  "scripts": {
+    "build": "tsc --build",
+    "test": "vitest",
+    "bundle": "esbuild --bundle --format=esm --outfile=dist/bundle.js index.ts"
```

Comment:
I'm not sure if we should ship with this sort of thing, but very useful during development for manual testing to have a bundler. Not sure common patterns for this in the JS world.